api/instance/v1/security_group_utils.go

```diff
@@ -97,13 +97,16 @@ type UpdateSecurityGroupRuleRequest struct {
 	SecurityGroupID     string   `json:"-"`
 	SecurityGroupRuleID string   `json:"-"`
 
-	Protocol     *SecurityGroupRuleProtocol  `json:"protocol,omitempty"`
-	Direction    *SecurityGroupRuleDirection `json:"direction,omitempty"`
-	Action       *SecurityGroupRuleAction    `json:"action,omitempty"`
-	IPRange      *string                     `json:"ip_range,omitempty"`
-	DestPortFrom *uint32                     `json:"dest_port_from,omitempty"`
-	DestPortTo   *uint32                     `json:"dest_port_to,omitempty"`
-	Position     *uint32                     `json:"position,omitempty"`
+	Protocol  *SecurityGroupRuleProtocol  `json:"protocol"`
+	Direction *SecurityGroupRuleDirection `json:"direction"`
+	Action    *SecurityGroupRuleAction    `json:"action"`
+	IPRange   *string                     `json:"ip_range"`
+	Position  *uint32                     `json:"position,omitempty"`
+
+	// If set to 0 DestPortFrom will be remove
+	DestPortFrom *uint32 `json:"dest_port_from"`
+	// If set to 0 DestPortTo will be remove
```

Comment:
1. "If set to 0, DestPortFrom will be removed."
2. It could be interesting to document a bit more this field